### PR TITLE
Creating popup window

### PR DESCRIPTION
**Pull request by tacettinkustu**
In this activity, I implemented a popup window that includes the project details in both desktop and mobile. In order to do that, I stored the information about my projects in a JavaScript object, then I added them dynamically on page load.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -195,6 +195,104 @@ li a:hover {
...
+  width: 90%;
+  height: 90%;
+  left: 5%;
+  top: 5%;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+}
+
```
Comment (rloterh) [inline style.css:230]: - [x] Your mobile menu doesn't display a blurred background when the hamburger is clicked, as shown in the Figma design. This makes the menu hard to read. Please put proper background blur to your mobile menu. You can use the [filter CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/filter) for this
- [x]  Also, the right margin of your mobile menu appears to wider than the left margin. This makes the design appear uncentered.  Let's ensure to center the content of our mobile menu.

Here's your design:
image

Here's the original design:
image

**Review by rloterh — CHANGES_REQUESTED** [review]
# Changes Required! ♻️

Hi Willy and Tacettin and Usman 👋 👋, I'm Robert, your reviewer for this PR.

I must commend you for the nice work done👏.  You nailed almost all of the specifications of this project right. There some necessary changes we need to implement to improve this project.

## Good points 👍
No Linters errors ✔️
Good use of HTML semantics ✔️


## Aspects to improve ♻️

_Check the comment(s) under the review._


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (tacettinkustu) [reply style.css:230]: Thank you for your reviews.

Comment (descholar-ceo) [reply style.css:230]: :+1: Addressed correctly :heavy_check_mark:

**Review by descholar-ceo — APPROVED** [review]
Hi @tacettinkustu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rloterh** [comment]
**Hello Tacettin and Usman!**

We have been discussing this project and common mistakes with the entire Code Reviewers team. After that, we realized that we have skipped one important requirement.

Your project is still approved but please make sure that **the projects list is not a hardcoded HTML code but instead it is loaded dynamically**. The goal is to have all the projects listed in your portfolio stored in one place. So, when you need to add something new you only need to add it to the array of objects.

We're sorry we couldn't point that out to you during the review. Kindly note, this was not on purpose 😊 


Happy coding! 🍷 💻 
Cheers!

_Please feel free to leave a comment if something is not clear._

**Pull request merged**